updatecli.d/appVersion.tpl

```diff
@@ -0,0 +1,24 @@
+name: Charts appVersion update
+
+sources:
+  {{- range $id, $opt := .charts }}
+  {{- if and ($opt) (index $opt "noAppVersion") }}{{ continue }}{{ end }}
```

Comment:
need to have a quick chat about this because the final version of the script is not really helm-aware by choice, see https://github.com/Alfresco/alfresco-build-tools/blob/master/scripts/gen-index-yml/gen_index_yml.py